spot_ros2_control/launch/spot_ros2_control.launch.py

```diff
@@ -178,6 +188,44 @@ def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
             namespace=spot_name,
         )
     )
+    # Optionally launch extra nodes for state and image publishing if on the robot
+    if on_robot:
+        # launch image publishers
+        ld.add_action(
+            IncludeLaunchDescription(
+                PythonLaunchDescriptionSource(
+                    [
+                        PathJoinSubstitution(
+                            [FindPackageShare("spot_driver"), "launch", "spot_image_publishers.launch.py"]
+                        )
+                    ]
+                ),
+                launch_arguments={
+                    key: LaunchConfiguration(key) for key in ["config_file", "spot_name"] + IMAGE_PUBLISHER_ARGS
+                }.items(),
+                condition=IfCondition(LaunchConfiguration("launch_image_publishers")),
+            )
+        )
+        # launch state publisher node (useful for publishing odom & other statuses)
+        ld.add_action(
+            Node(
+                package="spot_driver",
+                executable="state_publisher_node",
+                output="screen",
+                parameters=[config_file, {"spot_name": spot_name}],
+                namespace=spot_name,
+            )
+        )
+        # launch object sync node (for fiducials)
+        ld.add_action(
+            Node(
+                package="spot_driver",
+                executable="object_synchronizer_node",
+                output="screen",
+                parameters=[config_file, {"spot_name": spot_name}],
+                namespace=spot_name,
+            )
+        )
```

Comment:
I didn't add an optional flag for these like with the image publishers since they are pretty minimal and don't have the same issues with bandwidth, open to discussion if anyone thinks these should not be launched by default.